website/server/libs/cron.js

```diff
@@ -347,7 +393,7 @@ export function cron (options = {}) {
 
   // Login Incentives
   user.loginIncentives++;
-  user.addNotification('LOGIN_INCENTIVE', {});
+  awardLoginIncentives(user);
```

Comment:
As long as it's before the line where the user is saved it doesn't really matter if it's before or after